.github/workflows/docker.yaml

```diff
@@ -85,12 +85,17 @@ jobs:
           docker run --rm inventree-test invoke --list
           docker run --rm inventree-test gunicorn --version
           docker run --rm inventree-test pg_dump --version
+          docker run --rm inventree-test test -f /home/inventere/init.sh
+          docker run --rm inventree-test test -f /home/inventere/tasks.py
+          docker run --rm inventree-test test -f /home/inventere/gunicorn.conf.py
```

Comment:
possible typos
```suggestion
          docker run --rm inventree-test test -f /home/inventree/init.sh
          docker run --rm inventree-test test -f /home/inventree/tasks.py
          docker run --rm inventree-test test -f /home/inventree/gunicorn.conf.py
```

Reply:
Intentional to check that the checks actually work :)

Reply:
Ah, that makes sense. I was really confused about how you would typo the project name